plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -177,6 +180,36 @@ public Database getDatabase(String dbName)
         return client.getDatabase(dbName);
     }
 
+    @Override
+    public Map<String, List<String>> getAllTables()
+            throws TException
+    {
+        return parseTableMetadata(client.getTableMeta("*", "*", ImmutableList.of()));
+    }
+
+    @Override
+    public Map<String, List<String>> getAllViews()
+            throws TException
+    {
+        return parseTableMetadata(client.getTableMeta("*", "*", ImmutableList.of("VIEW")));
```

Comment:
Should we use `VIRTUAL_VIEW` here ?

Reply:
Although some PTs failed and it looks it might be related to this.
I'll reproduce locally and dig in more into this